### PR TITLE
Now you can forget connections

### PR DESCRIPTION
**Pull request by davidkelly**
This addresses [this bug](https://highfidelity.fogbugz.com/f/cases/3935/Add-ability-to-forget-a-connection).  Main thing here is we now can forget a connection, and the connections tab now looks a bit different when selected.

Note we have images for forgetting and info that already existed.  They are (IMHO) less than perfect.  Good enough?

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 06bd9c97bf160e45d29414b54016e47f6f172414 available at: https://gist.github.com/e6f6e26243f8fa9d55ac76bf98d2665f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10138-06bd9c9.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10138-06bd9c9.dmg

File: scripts/system/pal.js [hunk 1]
```diff
@@ -269,6 +269,21 @@ function fromQml(message) { // messages are {method, params}, like json-rpc. See
         getConnectionData();
         UserActivityLogger.palAction("refresh_connections", "");
         break;
+    case 'removeConnection':
+        connectionUserName = message.params;
+        request({
+            uri: METAVERSE_BASE + '/api/v1/user/connections/' + connectionUserName,
+            method: 'DELETE'
+        }, function (error, response) {
+            print(JSON.stringify(response));
```
Comment (howard-stearns) [inline scripts/system/pal.js]: debug code? remove?

File: interface/resources/qml/hifi/NameCard.qml [hunk 2]
```diff
@@ -346,6 +330,93 @@ Item {
...
+        anchors.verticalCenter: nameCardConnectionInfoImage.verticalCenter
+        anchors.leftMargin: 5
+        verticalAlignment: Text.AlignVCenter
+        text: "Info"
+        color: hifi.colors.baseGray
+    }
+    StateImage {
+        id: nameCardRemoveConnectionImage
+        visible: selected && !isMyCard && pal.activeTab == "connectionsTab"
+        imageURL: "../../images/info-icon-2-state.svg" // PLACEHOLDER!!!
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/NameCard.qml]: Did you see my comments on the ticket re GlyphButton? Or where they not productive?

Comment (davidkelly) [inline interface/resources/qml/hifi/NameCard.qml]: No - you had comments about GlyphButton?

File: scripts/system/pal.js (hunk 1 above)
Comment (davidkelly) [inline scripts/system/pal.js]: :) copied from another spot.  I'll remove both

File: interface/resources/qml/hifi/NameCard.qml (hunk 2 above)
Comment (davidkelly) [inline interface/resources/qml/hifi/NameCard.qml]: I just copied this pattern from how it was being done before -- would be interested in your comments...

Comment (davidkelly) [inline interface/resources/qml/hifi/NameCard.qml]: Oh ha now I see it.  I didn't find an appropriate one in my first look, will look again.  Thx

**Comment by hifi-gustavo** [comment]
Build console output for commit: a92162dea9f8ba90af6ad755840f8be699ca95ff available at: https://gist.github.com/e6f6e26243f8fa9d55ac76bf98d2665f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10138-a92162d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10138-a92162d.dmg

**Comment by davidkelly** [comment]
Also fixes: [this](https://highfidelity.fogbugz.com/f/cases/3985/PAL-Cannot-anchor-to-an-item-that-isn-t-a-parent-or-sibling) and [this](https://highfidelity.fogbugz.com/f/cases/3986/PAL-depends-on-non-NOTIFYable-properties)

**Comment by davidkelly** [comment]
Test Plan

Make some connections.  
Go to connections tab
Select someone, observe and compare to wireframes in [here](https://highfidelity.fogbugz.com/f/cases/3935/Add-ability-to-forget-a-connection).
Hit info, make sure it goes where you expect
Hit forget, make sure it really does disconnect them
Open interface logs, verify [this](https://highfidelity.fogbugz.com/f/cases/3985/PAL-Cannot-anchor-to-an-item-that-isn-t-a-parent-or-sibling) and [this](https://highfidelity.fogbugz.com/f/cases/3986/PAL-depends-on-non-NOTIFYable-properties) no longer are in the logs
:beers:

**Comment by hifi-gustavo** [comment]
Build console output for commit: 85c78da83959fb3381776158f3b0ac8502f036a7 available at: https://gist.github.com/e6f6e26243f8fa9d55ac76bf98d2665f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10138-85c78da.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10138-85c78da.dmg

**Comment by howard-stearns** [comment]
Hmmm. Can't open connections tab. "This is taking longer than usual...."

**Comment by howard-stearns** [comment]
QA fail.  It doesn't open because it gets:  file:///C:/Program Files/High Fidelity - PR10138/resources//qml/hifi/Pal.qml:1393: TypeError: Cannot call method 'toString' of undefined"

**Comment by howard-stearns** [comment]
The above fail is because of https://highfidelity.fogbugz.com/f/cases/4006/Connections-tab-never-loads, not this PR

**Pull request merged**